### DISCUSSION TITLE
Thank You

### DISCUSSION THREAD

**Discussion by trk2** [OpenSageTV/Community#120] [Gemstone]
Jusjoken, I wanted to express my gratitude for your work on Gemstone.  I know you didn't do it alone and the Phoenix team should be included for their work too, but you seem to be the last of those developers left.  I cannot express how much I prefer the UI you developed over the standard media wall format which dominates everything.  The media wall is the equivalent of someone who stores every file on their desktop and browses folders in icon view.  It's terrible.  In contrast, with Gemstone I can have a clean and elegant interface that better represents the beauty and pageantry of why we love television and movies.  My thanks.

![image](https://github.com/OpenSageTV/Community/assets/15076309/19de8c15-6269-4b4f-84d2-fca7355b25c1)

![image](https://github.com/OpenSageTV/Community/assets/15076309/e5e542a0-2057-4ce2-bd21-7fb2c5a90924)

**Comment by jusjoken** [comment]
Your welcome.  I enjoyed making it and interacting with other developers and users.  I wish the community was stronger but glad to see some of us die hards still around. 

K

**Comment by Sagetvmaster** [comment]
I think alot of users still exist, but when Narflex killed the good forum, many lost interest with Github?

**Comment by TGsell1** [comment]
GitHub is a challenge but lots of us are still trying!Sent from my iPhoneOn Mar 16, 2024, at 8:58 PM, Sagetvmaster ***@***.***> wrote:﻿
I think alot of users still exist, but when Narflex killed the good forum, many lost interest with Github?

**Comment by KryptoNyte-Sage** [comment]
Agree 1000% !!  I use Gemstone on every client I have.

Still alive and kicking.

**Comment by wayner9** [comment]
Me too, but I am worried that my days using Sage may be coming to an end due to changes in my cable infrastructure as my existing boxes will soon be EOLed and it may be too much hassle to get tuning working on new types of boxes, plus needing HDMI to Component adapters.
